open-repo.sh

```diff
@@ -0,0 +1,27 @@
+#!/data/data/com.termux/files/usr/bin/bash
+
+# Variable 'GIT_REPO_PATH' will be set by $HOME/repo.conf or by providing an argument (e.g. `open-repo.sh ~/storage/shared/git/notes`)
+source $HOME/repo.conf
```

Comment:
Could check if `repo.conf` exists and only source if necessary. Or even better: If the file exists then use it (and ensure that `GIT_REPO_PATH` is now set), but if the file does not exist then ensure that the user has passed in an argument (i.e. `$1`) and use that for `GIT_REPO_PATH` instead.